sympy/physics/continuum_mechanics/arch.py

```diff
@@ -639,3 +644,347 @@ def solve(self):
 
         self._axial_force = axial_force
         self._shear_force = shear_force
+
+    @doctest_depends_on(modules=('numpy',))
+    def draw(self):
+        """
+        This method returns a plot object containing the diagram of the specified arch along with the supports
+        and forces applied to the structure.
+
+        Examples
+        ========
+
+        >>> from sympy import Symbol
+        >>> t = Symbol('t')
+        >>> from sympy.physics.continuum_mechanics.arch import Arch
+        >>> a = Arch((0,0),(40,0),crown_x=20,crown_y=12)
+        >>> a.apply_load(-1,'C',8,150,angle=270)
+        >>> a.apply_load(0,'D',start=20,end=40,mag=-4)
+        >>> a.apply_load(-1,'E',10,t,angle=300)
+        >>> p = a.draw()
```

Comment:
![Arch_2](https://github.com/user-attachments/assets/3e5cb322-c440-4f46-adb9-d30e3ad0cce8)

- The arch is sometimes skewed towards the bottom maybe try getting the centering better. 
- Add units for the forces and maybe bold the magnitudes
- The arrows for the distributed loads seem too close
- How about filling the arch body and member like we do in trusses

Reply:
The arch body and skew appear as such cause I am plotting two equation `self._shape_eqn` and `self._shape_eqn-0.05*lim` which gives us two lines, there is not option to give a line width . I will be fixing the arrow closeness. I would like to know if there is some way to plot curves to have fill or width.

Reply:
![image](https://github.com/user-attachments/assets/f43672d4-cc51-4ad4-ba8e-20d6719572dd)
The arrows are now at a more suitable distance.